lib/jvertica.rb

```diff
@@ -54,7 +60,10 @@ def closed?
   end
 
   def close
-    @connection.close && @closed = true
+    @connection.close
+  ensure
+    @connection = nil
+    @closed = true
```

Comment:
Should ensure to set `@closed = true` even if `close` failed (Failing `close` means it is already closed usually)